.github/workflows/build_and_test.yml

```diff
@@ -279,3 +279,22 @@ jobs:
     #  run: clang -v
     #- name: Windows Test
     #  run: C:\Rust\.cargo\bin\cargo.exe test --verbose
+
+  freebsd:
+    runs-on: macos-12
+    name: Simple build in FreeBSD
+    steps:
+    - uses: actions/checkout@v2
+    - name: Test in FreeBSD
+      id: test
+      uses: vmactions/freebsd-vm@v0
+      with:
+        usesh: true
+        prepare: |
+          pkg install -y curl
+          curl https://sh.rustup.rs -sSf | sh -s -- -y
+
+        run: |
+          freebsd-version
```

Comment:
if you wish, you can install one of the llvm/clang version (the system clang does not have llvm dev headers) and points LLVM_CONFIG to /usr/local/bin/llvm-config-<version>

Reply:
`/usr/local/bin/llvm-config-14` doesn't seem to work

Reply:
Ah, my `add` command may have been wrong

Reply:
ah add and install are aliases, sorry I realise I made a typo most likely :-) seems you found out